### ISSUE TITLE
RPi Zero W2 substitution?

### ISSUE THREAD

**Issue by etknorr** [Harvey71/emdr#8]
Hey, thanks for this great guide! A friend asked me about making one of these setups and this is exactly the path I was going to start heading down. I have some raspberry pi's laying around which I'm not keen on giving up given the state of the Pi market... I'm curious if you had insight on if a Pi Zero W2 would be up to this task? A list of concerns arises:

Concerns:
* Processing power for the Pygame app
  * I think this would be okay for a relatively simple app that it is.
* Power consumption
  * Thinking specifically of the buzzers.. thoughts?
  * Likely would be fine, even if I had to power the Teensy through external power (not through the Pi)
* Obvious limitation of USB ports
  * Potentially use an OTG cable --> USB Hub (???)
  * Could use only one device at a time (either buzzers or light bar, not both)

**Comment by Harvey71** [comment]
I share your concerns with replacing the Pi by a Pi zero, but partially for different reasons:

- The devices (Teensy) are powered by the 5V line of the USB bus, which is more or less diectly connected to the external power supply. I recommend a 5V/1A power adapter - then you should be safe.
- The Pi zero only has one USB port. If you only use either light bar or buzzers at the same time, that should not bother you. If you plan to use both at the same time, a USB Hub would work. But that would be nasty cabling on the table. I would rather spend the money for a Raspberry Pi 3/4 than for the hub.
- Processing power should not be an issue. But maybe the 512MB RAM is a bit tight. You would have to try that out.
- My major concern is the different form factor. Thus you cannot use the out-of-the box casing for the display. Worst case you have to tinker (3D-print) a casing on your own.

TL;DR: I do not recommend using the RPi Zero. If you want to have a standalone controller, go with a Raspberry Pi 3/4 or drop-in-replacement. But I want to emphasize, that if the user has a PC or Laptop in their office, they can attach the light bar and buzzers directly to it. The controller software will run on Windows or MacOS in a window. Then you would not need any Raspberry at all.

If you sill want to use the RPi Zero, I wish you good luck and look forward to your feedback about your success. :-)
